docs/handlers.rst

```diff
@@ -19,22 +19,23 @@ If the bot receives a message that is exactly equal to `ping` the response will
 
 Bottery has the following handlers available:
 
+
```

Comment:
why two lines?

Reply:
Two lines between sections. I changed it a bit, is it better?